### PR TITLE
Upgrade thelounge 2.6.0

### PR DESCRIPTION
**Pull request by lcarva**
And set npm package name properly.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "electron-quick-start",
+  "name": "electron-lounge",
```
Comment (xPaw) [inline package.json]: Call it `thelounge-electron` please.

Comment (lcarva) [reply package.json]: Done.

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "electron-quick-start",
+  "name": "thelounge-electron",
```
Comment (astorije) [inline package.json]: Are names of this package and repo are reversed for a particular reason?

Comment (xPaw) [reply package.json]: Repo needs renaming anyway.

**Pull request merged**